plugins/inputs/kafka_consumer/kafka_consumer.go

```diff
@@ -43,6 +43,7 @@ type KafkaConsumer struct {
 	BalanceStrategy        string          `toml:"balance_strategy"`
 	Topics                 []string        `toml:"topics"`
 	TopicTag               string          `toml:"topic_tag"`
+	FetchMessageBytes      int32           `toml:"fetch_message_bytes"`
```

Comment:
How about making this a [`config.Size`](https://github.com/influxdata/telegraf/blob/bd516ae587a27d040b457c593e3d5fa6aaf40269/config/types.go#L15) so you can specify human-readable sizes like `1MB` or `1MiB`?